sdk/loadtesting/azure-developer-loadtesting/tests/test_run_ops.py

```diff
@@ -165,7 +163,6 @@ def test_stop_test_run(self, loadtesting_endpoint):
                 "displayName": "My New Load Test Run from PyTest",
             }
         )
-        assert run_poller.get_initial_response() is not None
```

Comment:
why are tests of the returned poller removed? we should call `.result()` on the returned object to get the final lro result, and do checks on the final object to make sure it's valid.

Reply:
Hi @iscai-msft, this is intentionally, this test is to test stop test run functionality!